spec/license_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+require 'rakelib/default_plugins'
+
+describe "Project licenses" do
+
+  let(:expected_licenses) {
```

Comment:
We should document here (or link to a place) the licenses we're expecting to match with these regular expressions, I'm guessing:

Apache License 2.0
BSD License
MIT license
..?